### ISSUE TITLE
Running OpenMHA without the bluetooth controller

### ISSUE THREAD

**Issue by JohnAbarro** [m-r-s/hearingaid-prototype#6]
Is it possible to install OpenMHA on pi without setting up the bluetooth controller? It appears to be method for directly adjusting some parameters, but for the configuration, we plan to make any sort of adjustments through wifi connection only. I would like to know if it will run without the bluetooth script. Thank you :)

**Comment by m-r-s** [comment]
Yes, of course. We added it just to demonstrate the possibilities.

The [Bluetooth control script](https://github.com/m-r-s/hearingaid-prototype/blob/0.8/tools/control/actions.sh) by default looks for the device "/dev/input/js0" and interprets its output.
But if the device does not exist everything will work just fine.  
This purpose of this project is to provide a functional configuration which you can then adapt to your needs.
You can also remove [start.sh](Line 58 from https://github.com/m-r-s/hearingaid-prototype/blob/master/start.sh#L58) in your setup to avoid that the "Bluetooth script" ever interferes with your configuration.

May I ask what your plans with the prototype are?

**Comment by JohnAbarro** [comment]
We would like to implement a hearing aid algorithm using OpenMHA on pi for our undergraduate thesis project. As you can see, we have little experience with using raspberry pi or linux, but somehow we are able to get by through online resources/tutorials. We appreciate how well-documented and detailed everything is with OpenMHA (we've done some perusing with the manuals), we only struggle with unfamiliarity. Thank you for being responsive and active as well! :)

**Comment by m-r-s** [comment]
Cool, I like that you want to use it in a scientific context :)
However, I think this could be very challenging for an undergraduate thesis project.

You have to distinguish **this** project ([https://github.com/m-r-s/hearingaid-prototype](https://github.com/m-r-s/hearingaid-prototype)) and the openMHA project ([https://github.com/HoerTech-gGmbH/openMHA](https://github.com/HoerTech-gGmbH/openMHA)).
**This** project only wants to demonstrate how far one can come with the right free and open source software tools and enable others to achieve the same, but the openMHA software is not developed here.
We are only users.

**Comment by JohnAbarro** [comment]
It is quite a big undertaking to use the software for the algorithm implementation, especially since we're all new to every single part of what we are studying. There's a bunch to cover in just months, and we've barely scratched the surface for weeks. It still seems possible to do, though. 

If I understood the difference correctly, the OpenMHA is the main open source software project for hearing aids which could be used on computers running linux, mac os, or even windows, and the hearingaid-prototype project uses this software in a way that it is configured for the raspberry pi setup shown. I'll take note of the difference when referencing the two projects.

**Comment by suaefar** [comment]
Yes, correct. This project only shows as an example how openMHA could be used in a mobile setup.
It also aims to motivate and help those who want to build something similar.
